Docs/Book/Cookbook.md

```diff
@@ -409,6 +409,8 @@ The code:
     fp = numpy.zeros((1,))
     DataStructs.ConvertToNumpyArray(AllChem.GetMorganFingerprintAsBitVect(m5, 2), fp)
 
+    # Reshape to a matrix for one sample
```

Comment:
I don't think it's that important; I just find reshape to be a somewhat confusing method.
I was just checking to see if that's just me